source/agora/node/Runner.d

```diff
@@ -73,3 +76,46 @@ public FullNode runNode (Config config)
     listenHTTP(settings, router);
     return node;
 }
+
+/*******************************************************************************
+
+    Set Vibe.d log level according to the configuration's log level
+
+    This is used to ensure we get the right amount of information from Vibe.d
+    Since a log level is the "minimum accepted" level, some level values might
+    not match. For example, if we want Vibe.d's "diagnostic" to be part of
+    the output when we set the loglevel to info, then that's what we must pass
+    to Vibe's `setLogLevel` (and since diagnostic < info, it will include the
+    latter too).
+
+    Params:
+        level = The level at which we want to set the logger
+
+*******************************************************************************/
+
+private void setVibeLogLevel (ILogger.Level level) @safe
+{
+    import vibe.core.log;
+
+    final switch (level)
+    {
+    case ILogger.Level.Trace:
+        setLogLevel(LogLevel.trace);
+        break;
+    case ILogger.Level.Info:
+        setLogLevel(LogLevel.diagnostic);
```

Comment:
Why?  Isn't that `LogLevel.info`?

Reply:
Look at the comment on the function

Reply:
Vive.d LogLevel
diagnostic, /// Extended user information (e.g. for more detailed error information)
info,       /// Informational message for normal user education
I see.